### PR TITLE
Seed initial admin

### PR DESCRIPTION
**Pull request by VILLAN3LL3**

### CODE REVIEW

File: Orso.Arpa.Api/Controllers/AuthController.cs [hunk 1]
```diff
@@ -31,9 +31,8 @@ public AuthController(IAuthService authService)
         [HttpPost("login")]
         [AllowAnonymous]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
-        [ProducesDefaultResponseType]
+        [ProducesResponseType(typeof(ErrorMessage), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ErrorMessage), StatusCodes.Status401Unauthorized)]
```
Comment (euphi967) [inline Orso.Arpa.Api/Controllers/AuthController.cs:35]: there are other _error_ responses in the whole API specification of the backend that lack the `typeof(ErrorMessage),` parameter. Please tidy up mid-term.

**Review by euphi967 — APPROVED** [review]
All changes are reasonable.
Please see my comment for mid-term tidy-up.

File: Orso.Arpa.Api/Controllers/AuthController.cs (hunk 1 above)
Comment (VILLAN3LL3) [reply Orso.Arpa.Api/Controllers/AuthController.cs:35]: @euphi967 Thanks for reviewing. I checked all controllers, but did not find any other incomplete error response types. Can you give me a clue where exactly you noticed this? However, I found and fixed another bug (see commit). Can you please check it and approve it again? Thank you!

**Review by euphi967 — APPROVED** [review]

File: Orso.Arpa.Api/Controllers/AuthController.cs (hunk 1 above)
Comment (euphi967) [reply Orso.Arpa.Api/Controllers/AuthController.cs:35]: fair point :-). I came from reviewing the `swagger.json` for _schema/$ref_ and I detected several 4xx responses with no _schema/$ref_ tag. Looking again showed, that only the 403 responses are lacking the _schema/$ref_.

**Review by euphi967 — APPROVED** [review]
_>However, I found and fixed another bug (see commit). Can you please check it and approve it again?_ 
Yeah! I did a grep for `typeof(ErrorMessage), StatusCodes.Status2` to plausibilize that this was the only one mistake.

File: Orso.Arpa.Api/Controllers/AuthController.cs (hunk 1 above)
Comment (euphi967) [reply Orso.Arpa.Api/Controllers/AuthController.cs:35]: how will we deal with those 403s? No need to specify _schema/$ref_ ?

**Pull request merged**